hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/autoReloadPanel/autoReloadPanel.tsx

```diff
@@ -27,6 +27,10 @@ import moment from 'moment';
 interface IAutoReloadPanelProps extends RouteComponentProps<object> {
   onReload: () => void;
   lastUpdated: number;
```

Comment:
Maybe we could do a small refactoring here, like you changed the text from "updated" to "refreshed" at the already existing timestamp on the page.
```suggestion
  lastRefreshed: number;
```

Reply:
Done Changes as Suggested by You.